### ISSUE TITLE
[Accessibility]: Search results page misc mobile bugs

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#34728]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34190

### Problem description

- [ ] "Expand all" button has flashing tooltip at widths <326px (fix so it is not flashing)

https://user-images.githubusercontent.com/206864/166076374-98ec6c8c-ef43-4407-b442-6506bc1d970f.mp4

- [ ] Symbol icon disappears if symbol name too long (icons should always be visible)
![image](https://user-images.githubusercontent.com/206864/166076413-d4267e0e-0c5e-4a33-80a9-66a229d9f829.png)

### Expected behavior

Issues are fixed

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.

**Comment by gitstart-sourcegraph** [comment]
Hi @limitedmage for last requirement: `Search result repo and path/commit message completely cut off on narrow screens. This happens for all result types.`
Want to confirm that we are looking to have a new layout that would not involve having to scroll result `left` or `right`, on mobile?

**Comment by limitedmage** [comment]
@gitstart-sourcegraph Correct, we need to wait for @sourcegraph/design to give us a new design to implement. Please continue to fix the other issues. I'll spin off the new design into a separate issue.

**Comment by limitedmage** [comment]
@gitstart-sourcegraph I've spun off the search result header issue to #36322 and have modified this issue to not include it. Please proceed fixing the other issues.
